tests/phpunit/tests/theme/wpAddGlobalStylesForBlocks.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+require_once __DIR__ . '/base.php';
+
+/**
+ * Tests wp_add_global_styles_for_blocks().
+ *
+ * @group themes
+ *
+ * @covers ::wp_add_global_styles_for_blocks
+ */
+class Tests_Theme_WpAddGlobalStylesForBlocks extends WP_Theme_UnitTestCase {
+
+	/**
+	 * Test blocks to unregister at cleanup.
+	 *
+	 * @var array
+	 */
+	private $test_blocks = array();
+
+	public function tear_down() {
+		// Unregister test blocks.
+		if ( ! empty( $this->test_blocks ) ) {
+			foreach ( $this->test_blocks as $test_block ) {
+				unregister_block_type( $test_block );
+			}
+			$this->test_blocks = array();
+		}
+
+		parent::tear_down();
+	}
+
+	/**
+	 * @ticket 56915
+	 */
+	public function test_third_party_blocks_inline_styles_not_register_to_global_styles() {
```

Comment:
Should this be

```suggestion
	public function test_third_party_blocks_inline_styles_not_registered_to_global_styles() {
```

?